### DISCUSSION TITLE
Resize Image Distortion

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#7327] [Help]
### ImageMagick version

7.1.1-33

### Operating system, version and so on

OS : macOS Monterey, version : 12.5,  chip : Apple MI Pro

### Description

Im trying the resize for the below image to multiple dimensions.

I can see the distortion in the resized image.

[Original Image and Resized Image](https://drive.google.com/file/d/1kL57TrMSLfxWJdn0HqwWbAd3GcWWXCFC/view?usp=sharing)

**Commands used to resize the Image**

 magick originalImage -filter Lanczos -resize 1600x900 resizedImage.png

**Comment by fmw42** [comment]
Your link requires a login username and password.  Please fix that or repost without it.

**Reply by programmerarunkumar** [reply]
Changed the permission for the link. kindly check now

**Comment by fmw42** [comment]
I do not see anything wrong.  Could you explain in more detail what is the distortion issue with these images and perhaps point it out on the images.  What tool are you using to view them?  Might the viewer be causing your apparent visual distortion?

**Reply by programmerarunkumar** [reply]
Please zoom and check the original image and resized image . you can see that the quality is reduced in resized image

I can especially able to notice this in image which has shapes, numbers, drawings, flowcharts

**Comment by fmw42** [comment]
That happens when you resize using any interpolation method. You cannot make a perfect smaller reproduction of the high resolution original, especially when you resize so much.  You can try other filters such as catrom or possibly nearestneighbor, or point. One of those may produce resizing more to your liking.  Any resizing is a trade between sharpness and artifacts.  Please see 

https://imagemagick.org/Usage/resize/
https://imagemagick.org/Usage/filter/
https://imagemagick.org/Usage/filter/nicolas/
